addons/mail/static/src/models/channel_member_list_view.js

```diff
@@ -1,66 +1,73 @@
 /** @odoo-module **/
 
 import { registerModel } from '@mail/model/model_core';
-import { one } from '@mail/model/model_field';
-import { clear, insertAndReplace, replace } from '@mail/model/model_field_command';
+import { attr, one } from '@mail/model/model_field';
+import { clear, replace } from '@mail/model/model_field_command';
 
 registerModel({
     name: 'ChannelMemberListView',
     identifyingFields: [['chatWindowOwner', 'threadViewOwner']],
+    lifecycleHooks: {
+        _created() {
+            this.channel.fetchChannelMembers();
+        },
+    },
     recordMethods: {
+        /**
+         * Handles click on the "load more members" button.
+         */
+        async onClickLoadMoreMembers() {
+            this.channel.fetchChannelMembers();
+        },
         /**
          * @private
-         * @returns {FieldCommand}
+         * @returns {Boolean}
```

Comment:
We usually use lower case to document primitive types: `Boolean` ➡️ `boolean`.
Plus, this function can now also return a FieldCommand:
```suggestion
         * @returns {boolean|FieldCommand}
```